meet-data/kpf/islandsmeistaramot-unglinga-og-oldunga-i-kraftlyftingum-2008/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+KPF,2008-02-23,Iceland,,Reykjavík,Íslandsmeistaramót unglinga og öldunga í kraftlyftingum
```

Comment:
I think these `MeetPath` entries are too unwieldy also, just like with the SVNL. Any objection to using our normal numbering system for short URLs?

Reply:
No problem, will switch them over when I have a minute.